apps/examples/timer/timer_main.c

```diff
@@ -146,12 +159,50 @@ static pthread_addr_t timer_thread(pthread_addr_t arg)
 		goto error;
 	}
 
+#ifdef CONFIG_EXAMPLES_TIMER_FRT_MEASUREMENT
+	/* Open the FRTimer device */
+	if (pargs->devno == 0) {
+		frt_dev = 1;
+	} else {
+		frt_dev = 0;
+	}
+	snprintf(path, _POSIX_PATH_MAX, TIMER_DEVNAME, frt_dev);
+	fprintf(stdout, "FRT Open %s\n", path);
+
+	frt_fd = open(path, O_RDONLY);
+	if (frt_fd < 0) {
+		fprintf(stderr, "ERROR: Failed to open Free Run Timer: %d\n", errno);
+		goto error;
+	}
+	ioctl(frt_fd, TCIOC_SETFREERUN, TRUE);
```

Comment:
what if the ioctl fails?

Reply:
It is just set the register value. ioctl always returns OK.